setup.py

```diff
@@ -44,16 +44,16 @@
     install_requires=['django-model-utils>=4.0.0,<4.1.0'],
     extras_require={
         'qa': [
-            'black<=19.10b0',
+            'black<=20.8b1',
             'flake8<=3.9',
             'isort~=5.0',
-            'readme-renderer~=26.0',
+            'readme-renderer~=28.0',
             'coveralls',  # depends on coverage as well
         ],
         'rest': [
-            'djangorestframework>=3.11,<3.12',
+            'djangorestframework~=3.12',
```

Comment:
are you sure this doesn't impact negatively the other modules?
I think we should test it on other modules before updating it here. If you want you can help me to open issues like this one in each module which uses DRF: https://github.com/openwisp/openwisp-radius/issues/194

See also https://github.com/openwisp/openwisp-utils/issues/164.

Reply:
I'll upgrade and test it on radius, and ping back here. :smile: